src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.Android.cs

```diff
@@ -107,7 +107,7 @@ private static int ParseGMTNumericZone(string name)
             {
                 return new TimeZoneInfo(id, TimeSpan.FromSeconds(0), id, name, name, null, disableDaylightSavingTime:true);
             }
-            if (name.StartsWith("GMT", StringComparison.Ordinal))
+            if (name.Length >= 3 && name[0] == 'G' && name[1] == 'M' && name[2] == 'T')
```

Comment:
I'm curious why this changed?

Reply:
It can be a perf win if we are able to delay loading ICU until after startup.  This change doesn't guarantee that, it just helps.

Reply:
Apparently, it's fairly easy to get to https://github.com/dotnet/runtime/blob/4da3eae7a54a4a6abeb59df4953b6de4c65b53ea/src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs#L19-L33 to trigger the load.